lib/MailQueueHandler.php

```diff
@@ -163,10 +164,16 @@ public function sendEmails($limit, $sendTime, $forceSending = false, $restrictEm
 		$deleteItemsForUsers = [];
 		$this->activityManager->setRequirePNG(true);
 		foreach ($affectedUsers as $user) {
+			if (isset($userEnabled[$user]) && $userEnabled[$user] === 'no') {
```

Comment:
For me this was set to `"false"`:

<img width="221" alt="bildschirmfoto 2017-09-28 um 17 49 38" src="https://user-images.githubusercontent.com/245432/30976592-7aa9d4f4-a475-11e7-94ea-3c1d8582f89c.png">

Reply:
🙌 to type safety ...

Reply:
argl

Reply:
Fixed